src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -67,7 +67,7 @@ tests:
     - label:
           "Step 2b: TH1 sends a subscription request action for FabricSceneInfo
           to the DUT."
-      PICS: S.S.A0007
+      PICS: S.S.A0002
       verification: |
```

Comment:
most of these are missing the actual verification steps, is GRL going to fill these in? Are they in the spreadsheet?

Reply:
Seems like an oversight, missing verification added.